### PR TITLE
Add ESC11

### PR DESCRIPTION
**Pull request by BlWasp**
This PR adds the AD CS ESC11 privesc. I have added the attack to the `web-endpoint.md` page since both ESC8 and 11 are NTLM relay attacks.

I have renamed the page to `unsigned-endpoints.md`, change all the references to the previous `web-endpoint.md` page across THR, and also change some indications about "ESC1 - ESC8" to "ESC1 - ESC14", since now there are 14 techniques.

### CODE REVIEW

File: ad/movement/ad-cs/unsigned-endpoints.md [hunk 1]
```diff
@@ -12,7 +12,11 @@ In [their research papers](https://posts.specterops.io/certified-pre-owned-d9591
 >
 > ([specterops.io](https://posts.specterops.io/certified-pre-owned-d95910965cd2#5c3c))
 
-This attack, like all [NTLM relay attacks](../ntlm/relay.md), requires a victim account to authenticate to an attacker-controlled machine. An attacker can coerce authentication by many means, see [MITM and coerced authentication coercion techniques](../mitm-and-coerced-authentications/). Once the incoming authentication is received by the attacker, it can be relayed to an AD CS web endpoint.
+Following this, [Sylvain Heiniger](https://twitter.com/sploutchy) from Compass Security has found a similar vulnerability on the AD CS RPC enrollment endpoint. As described in [his article](https://blog.compass-security.com/2022/11/relaying-to-ad-certificate-services-over-rpc/), RPC interfaces allows you to check the NTLM signature on each independently.
```
Comment (ShutdownRepo) [inline ad/movement/ad-cs/unsigned-endpoints.md:15]: RPC interfaces allow to check signature on each what?

File: ad/movement/ad-cs/unsigned-endpoints.md [hunk 2]
```diff
@@ -12,7 +12,11 @@ In [their research papers](https://posts.specterops.io/certified-pre-owned-d9591
...
+
+For certificate request purpose, the `MS-ICPR` (ICertPassage Remote Protocol) RPC interface is used. According to the [Microsoft documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-icpr/0c6f150e-3ead-4006-b37f-ebbf9e2cf2e7), packet privacy is enabled if the `IF_ENFORCEENCRYPTICERTREQUEST` flag is set (default configuration), meaning that NTLM relay attacks are not possible.
```
Comment (ShutdownRepo) [inline ad/movement/ad-cs/unsigned-endpoints.md:17]: Any way we can check if the target is vulnerable before trying to attack it?

File: ad/movement/ad-cs/unsigned-endpoints.md [hunk 3]
```diff
@@ -79,10 +85,62 @@ If web endpoints are enabled, switch to UNIX because at the time of writing (Oct
...
+{% endhint %}
+
+[Certipy](https://github.com/ly4k/Certipy) (Python) can be used to enumerate information regarding the certificate templates (EKUs allowing for authentication, allowing low-priv users to enroll, etc.) and identify a vulnerable RPC endpoint ([how to enumerate](./#attack-paths)).
+
+<pre class="language-python"><code class="lang-python"><strong># find ESC11-vulnerable CAs
+</strong><strong>certipy find -u "$USER@$DOMAIN" -p "$PASSWORD" -dc-ip "$DC_IP" -stdout | grep -B20 ESC11
+</strong><strong>
+</strong><strong># find and look through enabled templates for ones that could be used for authentication
+</strong>certipy find -u "$USER@$DOMAIN" -p "$PASSWORD" -dc-ip "$DC_IP" -stdout -enabled
+</code></pre>
```
Comment (ShutdownRepo) [inline ad/movement/ad-cs/unsigned-endpoints.md:113]: can you replace the HTML/GitBook formatting with what follows please?

```python
# insert code
```

File: ad/movement/ad-cs/unsigned-endpoints.md [hunk 4]
```diff
@@ -24,23 +28,25 @@ This allows for lateral movement, account persistence, and in some cases privile
...
+The certificate template flag (i.e. `--template`) can either be left blank (defaults to Machine or User whether relayed account name ends with `$`) or chosen among the certificate templates that fill the requirements.
 
 For instance, if the relayed principal is a domain controller, the `DomainController` template must be specified.
 {% endhint %}
 
-[Certipy](https://github.com/ly4k/Certipy) (Python) can be used to enumerate information regarding the certificate templates (EKUs allowing for authentication, allowing low-priv users to enroll, etc.) ([how to enumerate](./#attack-paths)).
+[Certipy](https://github.com/ly4k/Certipy) (Python) can be used to enumerate information regarding the certificate templates (EKUs allowing for authentication, allowing low-priv users to enroll, etc.) and identify enabled HTTP endpoint ([how to enumerate](./#attack-paths)).
 
 <pre class="language-python"><code class="lang-python"><strong># find ESC8-vulnerable CAs
 </strong><strong>certipy find -u "$USER@$DOMAIN" -p "$PASSWORD" -dc-ip "$DC_IP" -stdout | grep -B20 ESC8
```
Comment (ShutdownRepo) [inline ad/movement/ad-cs/unsigned-endpoints.md:52]: can you replace the HTML/GitBook formatting with what follows please?

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]
Below some change requests/questions, and we should be good to go, thank you!!

File: ad/movement/ad-cs/unsigned-endpoints.md (hunk 2 above)
Comment (BlWasp) [reply ad/movement/ad-cs/unsigned-endpoints.md]: The command check is already indicated in the ESC11 part 😄 like the ESC8

**Review by ShutdownRepo — APPROVED** [review]

**Pull request merged**